lib/Blockchain.ts

```diff
@@ -12,24 +12,24 @@ import {
   MempoolEntryResult,
   MempoolInfoResult,
   TxOutResult
-} from "bitcoin-com-rest"
-import { REST_URL } from "./BITBOX"
+} from "bitcoin-com-rest";
+import { REST_URL } from "./BITBOX";
 
 export class Blockchain {
-  public restURL: string
+  public restURL: string;
   constructor(restURL: string = REST_URL) {
-    this.restURL = restURL
+    this.restURL = restURL;
```

Comment:
I'm working on fixing this linting noise. I will submit a commit that removes these semicolons.